### PR TITLE
Ct 1167/030 statement tests conversion

### PR DESCRIPTION
**Pull request by VersusFacit**
### Description

Converting another test.

### Checklist

- [X] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [X] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [X] I have run this code in development and it appears to resolve the stated issue
- [X] This PR includes tests, or tests are not required/relevant for this PR
- [X] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR

### CODE REVIEW

File: tests/functional/statements/fixtures.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
-id,first_name,last_name,email,gender,ip_address
```
Comment (VersusFacit) [inline tests/functional/statements/fixtures.py:1]: Kind of amazing Git decided to label this a rename based on hunks.

File: tests/functional/statements/test_statements.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from tests.functional.statements.fixtures import (
+    models__statement_actual,
+    seeds__statement_actual,
+    seeds__statement_expected,
+)
+
+
+class TestStatements:
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
```
Comment (colin-rogers-dbt) [inline tests/functional/statements/test_statements.py:18]: nit: snake case

**Review by colin-rogers-dbt — APPROVED** [review]
LGTM

**Comment by VersusFacit** [comment]
To answer a question in Slack, these are testing our Jinja statements are running correctly.

**Pull request merged**